src/screens/MemberDetail/MemberDetail.tsx

```
@@ -464,16 +560,131 @@
+                  />
+                </Col>
+            </Row>
+            <Row className="mt-4">
+              <Col md={5}>
+                <label htmlFor="language" className="form-label" style={{color : '#495057'}} >
+                  {t('chooseLanguage')}
+                </label>
+                <button 
+                  className={`${styles.inputColor} small form-control d-flex align-items-center`}
+                  style={{ width: 'fit-content' , gap: "0.5rem", border:'solid', borderWidth:'2px', borderColor:'#ced4da' }}
+                >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)